cves/CVE-2010-0425.yml

```diff
@@ -97,8 +102,9 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
+  answer: The vulnerability was found by Brett Gervasoni of Sense of Security
+    who also proposed a fix for the vulnerability.
```

Comment:
Was there any exploits created for this specific vulnerability, or was it fixed before attackers could abuse it?

Reply:
From what I researched I saw no evidence of an exploit being created for this vulnerability so I believe that would mean it was fixed before attackers could abuse it. I added this information to the CVE in the latest commit.